lib/api_catalogue.rb

```diff
@@ -10,6 +12,28 @@ def self.from_csv(catalogue_csv:, organisation_csv:)
     new(apis: apis, organisations: organisations)
   end
 
+  def self.from_url(url)
+    V1AlphaFetcher.fetch(url)
+  rescue RuntimeError => e
```

Comment:
per comment elsewhere, if we had a specific error thrown, we could match on it, i.e. ignoring the `NoVersionSupported`